nncf/quantization/fake_quantize.py

```diff
@@ -246,6 +259,38 @@ def calculate_quantizer_parameters(
     return FakeQuantizeParameters(input_low, input_high, output_low, output_high, levels)
 
 
+def calculate_convert_parameters(
+    statistics: MinMaxTensorStatistic,
+    is_activation: False,
+    destination_type: str = "HF8",
```

Comment:
What do you think about using `enum` instead of `str`?

```
class DestinationType(Enum):
   HF8 = "HF8"
   BF8 = "BF8"
```

Reply:
Done as discussed.